PatientWebApplication/AppointmentMicroserviceApi/Model/DoctorAppointment.cs

```diff
@@ -0,0 +1,57 @@
+/***********************************************************************
+ * Module:  DoctorAppointment.cs
+ * Author:  Mladenka
+ * Purpose: Definition of the Class Patient.DoctorAppointment
+ ***********************************************************************/
+
+using AppointmentMicroserviceApi.Doctor;
+using AppointmentMicroserviceApi.Model;
+using AppointmentMicroserviceApi.Utility;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace AppointmentMicroserviceApi.Patient
+{
+    public class DoctorAppointment : Appointment
+    {
+        [ForeignKey("AppointmentId")]
+        public virtual List<Referral> referral { get; set; }
```

Comment:
Change name of property 'referral' into 'Referral', and change then in constructor.